PayCalculatorAPI/Program.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace PayCalculatorAPI
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                });
+    }
+}
```

Comment:
You're using .NET Core 3.1 at the moment, which is fine. However you could experiment with updating it to the latest version, .NET 5. That would let you cut this file down a bit by making it a [top level program](https://gunnarpeipman.com/csharp-top-level-programs/).

Reply:
Okay thank you I will do that today.

Reply:
The reason I have it at 3.1 is because the course I am following is using NET Core 3.1. But I don't mind updating it now or after I have completed the course.